domain/schema/model.go

```diff
@@ -34,6 +42,15 @@ func ModelDDL() *schema.Schema {
 		blockDeviceSchema,
 		changeLogTriggersForTable("block_device_machine", "machine_uuid", tableBlockDeviceMachine),
 		storageSchema,
+		secretSchema,
+		changeLogTriggersForTable("secret", "auto_prune", tableSecret),
```

Comment:
These trigger definitions need attention. They should facilitate the ability to watch for an entity to change in the given namespace. This one would be writing true/false as an identity.